buildenv/jenkins/variables/defaults.yml

```diff
@@ -29,6 +29,8 @@ openjdk_branch: 'openj9'
 #========================================#
 # Miscellaneous settings
 #========================================#
+specs: 'linux_ppc-64_cmprssptrs_le,linux_390-64_cmprssptrs,aix_ppc-64_cmprssptrs'
+sdks: '8,9'
 jdk_image_dir:
```

Comment:
I like the idea, but I wonder if might we ever hit a situation where a spec won't be performed for a release?

Reply:
If I understand correctly, an imaginary example: no aix for 8 combination. I think that's fine, we wouldn't have a Jenkins job to do such a thing. We still would maintain the amount of Jenkins jobs we have now, but instead of having a file per job we may only have 3/4 files to cover all the job types. I think what you are thinking of we would have to worry about in the Pipeline-Build-Test-All file.